contracts/core/Resolving/SportsAMMV2ResultManager.sol

```diff
@@ -456,6 +464,21 @@ contract SportsAMMV2ResultManager is Initializable, ProxyOwned, ProxyPausable, P
                 );
     }
 
+    function _exerciseLosingTickets(address[] memory _tickets, uint _numOfTicketsToExercise) internal returns (uint) {
```

Comment:
As a coding best practice, input method params should never change value during the method, so please declare a new return variable and initiate it with the value of _numOfTicketsToExercise